_posts/2016-05-25-a-simple-TDD-environment-in-haskell.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+name: simple-tdd-environment-haskell
+title: A simple TDD environment in Haskell
+date: 2016-05-25 00:01:00 +00:00
+author: Liam Griffin
+image:
+    src: /assets/img/custom/blog/2016-05-25-haskell.svg
+tags:
+- tdd
+- haskell
+---
+
+I recently tried the bowling kata in Haskell ([finished implementation here](https://github.com/Gryff/bowling-kata)), which caused me to find out how set up my environment to comfortably do TDD. Hopefully others might find this helpful to begin their journey with the language. Here’s what I used:
```

Comment:
`I recently implemented the [bowling kata in Haskell](https://github.com/Gryff/bowling-kata)). In the process, I found out how to set up my environment to comfortably do Test Driven Development. Hopefully, others might find this post helpful to begin their journey with the [Haskel](link to Haskel website) language. I used the following components:`